codechecker_lib/analyzer.py

```diff
@@ -134,6 +131,9 @@ def run_check(args, actions, context):
     end_time = time.time()
 
     with client.get_connection() as connection:
+        if os.path.exists(suppress_file):
+            client.send_suppress(context.run_id, connection, suppress_file)
+
```

Comment:
The idea here was that when a bug suppression is stored then we check the bug hash if it exists in the database so we don't store those twice. But since the intention is to keep the suppression from the source code then the suppressions from suppress file have to come later.